examples/with-firebase-authentication-serverless/next.config.js

```diff
@@ -0,0 +1,18 @@
+require('./env.js')
+
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE_BUNDLE === 'true',
+})
+
+const nextConfig = {
+  // Public, build-time env vars.
+  // https://nextjs.org/docs#build-time-configuration
+  env: {
+    FIREBASE_AUTH_DOMAIN: process.env.FIREBASE_AUTH_DOMAIN,
+    FIREBASE_DATABASE_URL: process.env.FIREBASE_DATABASE_URL,
+    FIREBASE_PROJECT_ID: process.env.FIREBASE_PROJECT_ID,
+    FIREBASE_PUBLIC_API_KEY: process.env.FIREBASE_PUBLIC_API_KEY,
+  },
+}
+
+module.exports = withBundleAnalyzer(nextConfig)
```

Comment:
Please remove the bundle analyzing plugin.

Reply:
Done.